### PR TITLE
fix(test): use demo app to test username-password flow in integration test

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

We should not assume the created user has an "admin" role name, so we should use the demo app instead because a user without an "admin" role name can not sign in to the admin console application.

## Testing

CI

### CODE REVIEW

**Review by IceHe — DISMISSED** [review]
LGTM. Better wait for another approval to double check

**Review by simeng-li — APPROVED** [review]
LGTM

File: packages/integration-tests/src/constants.ts
Comment (gao-sun) [inline packages/integration-tests/src/constants.ts:7]: if you imported schemas, use the const from schemas

**Review by IceHe — APPROVED** [review]

**Review by gao-sun — CHANGES_REQUESTED** [review]
please revisit your PR description

**Review by gao-sun — APPROVED** [review]

**Pull request merged**
